src/test/java/com/jcabi/ssh/SSHByPasswordTest.java

```diff
@@ -29,40 +29,28 @@
  */
 package com.jcabi.ssh;
 
-import com.google.common.io.Files;
 import com.jcabi.log.Logger;
 import java.io.ByteArrayOutputStream;
-import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.net.InetAddress;
 import java.net.ServerSocket;
-import java.util.Collections;
 import java.util.logging.Level;
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.io.input.NullInputStream;
 import org.apache.sshd.SshServer;
-import org.apache.sshd.common.NamedFactory;
 import org.apache.sshd.server.Command;
 import org.apache.sshd.server.CommandFactory;
 import org.apache.sshd.server.Environment;
 import org.apache.sshd.server.ExitCallback;
-import org.apache.sshd.server.PasswordAuthenticator;
-import org.apache.sshd.server.UserAuth;
-import org.apache.sshd.server.auth.UserAuthPassword;
-import org.apache.sshd.server.keyprovider.SimpleGeneratorHostKeyProvider;
-import org.apache.sshd.server.session.ServerSession;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
 import org.junit.Test;
-import org.mockito.Mockito;
 
 /**
  * Unit tests for {@link SSHByPassword}.
- * @todo #21:30min SSH server setup in this class is nearly identical to the
- *  one in {@link SSHTest}. It should be extracted in a separate reusable
- *  module.
+ *
```

Comment:
@prondzyn do we really need checkstyle suppression below ?
`@checkstyle ClassDataAbstractionCoupling (500 lines)`